Paint house.py

```diff
@@ -0,0 +1,62 @@
+#LC: yes, 256
+#problems : NA
+
+#recursive
+#TC: O(k*2^n) ,n is no of houses; levels in tree is 2*n, there are k such trees
+###TLE Error
+#SC: O(k*2^n) recursive call stacks
+class Solution:
+    def minCost(self, costs: List[List[int]]) -> int:
+        
+        def helper(costs, idx, color):
+            if idx == len(costs):
+                return 0
+            if color == 0:
+                return costs[idx][0] + min(helper(costs, idx+1, 1), helper(costs, idx+1, 2))
+            if color == 1:
+                return costs[idx][1] + min(helper(costs, idx+1, 0), helper(costs, idx+1, 2))
+            if color == 2:
+                return costs[idx][2] + min(helper(costs, idx+1, 0), helper(costs, idx+1, 1))
+            return None
+        
+        costsR = helper(costs, 0, 0)
+        costsB = helper(costs, 0, 1)
+        costsG = helper(costs, 0, 2)
+        
+        return min(costsR, min(costsB, costsG))
+    
+#DP
+#TC: O(n) ,n is no of houses; levels in tree is 2*n, there are k such trees
+###TLE Error
```

Comment:
Is this giving TLE then please share the TLE test case

Reply:
no , TLE was only for first one. I changed it now in my latest commit